src/GithubClientFactory.ts

```diff
@@ -0,0 +1,41 @@
+import {GithubClientConfigs} from './Config';
+import {Logger} from "./Logger";
+/* const { throttlingPlugin } = require("@octokit/plugin-throttling");
+const { retryPlugin } = require("@octokit/plugin-retry");
+
+ */
+const Octokit = require("@octokit/rest").plugin(require("@octokit/plugin-throttling")).plugin(require("@octokit/plugin-retry"));
+
+export class GithubClientFactory  {
+    octokitClient;
+    constructor(private readonly config: GithubClientConfigs, logger :Logger) {
```

Comment:
Je viens de réaliser que les commandes de tslint sont dans package.json, mais `tslint` n'est pas dans les `devDependencies` :(

`config` n'est pas utilisé en dehors du constructeur.

```suggestion
    constructor(config: GithubClientConfigs, logger: Logger) {
```